core/modules/widgets/link.js

```diff
@@ -153,7 +153,7 @@ LinkWidget.prototype.handleDragStartEvent = function(event) {
 			}
 			// Then the data
 			dataTransfer.clearData();
-			var jsonData = this.wiki.getTiddlerAsJson(this.to),
+			var jsonData = this.wiki.getTiddlerAsJson(this.to,true),
```

Comment:
Perhaps a more general modification to getTiddlerAsJson would be to pass an optional hashmap of fields that are superimposed over the fields of the specified tiddler (including `undefined` causing the field to be omitted from the JSON).